client-js/app/elements/labrad-plot.ts

```diff
@@ -181,6 +188,32 @@ export class Plot extends polymer.Base {
     }
   }
 
+  @observe('drawMode2D')
+  drawMode2DChanged(newMode: string, oldMode: string) {
+    switch (newMode) {
+      case 'dots':
+        this.$.dots.style.color = 'black';
+        this.$.rectfill.style.color = '#AAAAAA';
```

Comment:
This style on this line is for disabled toggle buttons, not the plot area. But I will make the plot area white; I think that's a good idea.